### PR TITLE
chore(readme): update styles and information

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

no

### Motivation / Use-Case

Change to the same style as webpack's readme.

### Breaking Changes

no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +31,7 @@ fast in-memory access to the webpack assets.
 7. [Attribution](#attribution)
 8. [License](#license)
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```
Comment (evilebottnawi) [inline README.md]: We should avoid using `html`, it is bad practice. Also let's do:
- Remove `Maintainers` section (it is outdated) and no need
- Don't move `div` after description

Comment (hiroppy) [reply README.md]: > * Don't move `div` after description

sorry, I want to get more details.

Comment (evilebottnawi) [reply README.md]: @hiroppy yep L21

**Comment by hiroppy** [comment]
updated

**Comment by hiroppy** [comment]
update

**Pull request merged**